feat(nuxt): Improve Nuxt server-side setup docs (#12102)
getsentry/sentry-docs: platform-includes/getting-started-verify/javascript.nuxt.mdx

### Server-side

Sentry can only be loaded on the server-side by being explicitly added via `--import`.
Check out the <PlatformLink to="/install/cli-import">`--import` CLI flag</PlatformLink> docs for setup instructions.

To verify that Sentry works on the server-side, add the following snippet on the server-side:

```js {tabTitle:Nitro} {filename:server/sentry-example-api.ts}
import { defineEventHandler } from '#imports';

export default defineEventHandler(event => {
  throw new Error("Sentry Example API Route Error");
});

```

> <Alert level="warning">

Reviewer: Not 100% sure, but do we need to make this an alert? What about instead, just making this regular text? This is part of the server-side verify step basically, isn't it?

  ```
  # Start your app after building your project with `nuxi build`
  node --import ./.output/server/sentry.server.config.mjs .output/server/index.mjs
  ```

<Alert level="warning">

>   Sentry server-side monitoring **does not work in development mode!**

Reviewer 2: ```suggestion
  Sentry server-side monitoring **doesn't work in development mode!**
```

>   In case you experience any issues with the server-side setup, check out <PlatformLink to="/troubleshooting">Troubleshooting</PlatformLink>
>   or read through the different <PlatformLink to="/install">installation methods</PlatformLink>.
> </Alert>

Reviewer 2: ```suggestion
  If you experience any issues with the server-side setup, check out <PlatformLink to="/troubleshooting">Troubleshooting</PlatformLink>
  or read through the different <PlatformLink to="/install">installation methods</PlatformLink>.
</Alert>
```
